src/visualization/types/Graph/options.tsx

```diff
@@ -49,6 +50,15 @@ interface Props extends VisualizationOptionProps {
 }
 
 export const GraphOptions: FC<Props> = ({properties, results, update}) => {
+  const [yAxisLabel, setYAxisLabel] = useState(properties.axes.y.label)
+  const [yAxisPrefix, setYAxisPrefix] = useState(properties.axes.y.prefix)
+  const [yAxisSuffix, setYAxisSuffix] = useState(properties.axes.y.suffix)
+
+  const groupKey = useMemo(
```

Comment:
just moving this `useMemo` hook higher up in the function so it's grouped with other hooks